infoUser/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace infoUser
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            User user = new User();              Console.WriteLine("Hey, let me ask you a little about yourself");
+
+            user.FirstName = user.AnswerQuestion("What's your first name?");              user.LastName = user.AnswerQuestion("What's your last name?");              user.FavAnimal = user.AnswerQuestion("What's your favorite animal?");              user.FavColor = user.AnswerQuestion("What's your favorite color?");              user.FavNumber = user.AnswerQuestionWithInteger("What's your favorite number?");              Console.WriteLine($"So you are { user.FirstName} { user.LastName} ");              Console.WriteLine($"Your spirit animal is a { user.FavAnimal}! ");
+
+            Console.Write($"And your lucky number is");
```

Comment:
This is fine, but doesn't have to have the dollar sign since there are no variables in it